src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
I understand that. But even now we are adding methods that are extensions to the service worker API, and not just the Notifications API. It is something users will likely want, and we're only talking about a one line feature detect at the end of the day.